packages/fury-adapter-remote/test/fury-test.js

```diff
@@ -256,7 +256,7 @@ describe('Adapter works with Fury interface', () => {
       fury.parse({ source: blueprintSource, mediaType: 'text/vnd.apiblueprint' }, (err, result) => {
         expect(result).to.be.undefined;
         expect(err).to.be.instanceof(Error);
-        expect(err).to.have.property('message', 'getaddrinfo ENOTFOUND some.stupid.non.existing.domain some.stupid.non.existing.domain:80');
+        expect(err.message).to.match(/ENOTFOUND some.stupid.non.existing.domain/);
```

Comment:
shouldn't the dots be escaped?

Reply:
I guess so, I'll do that. `.` in regex is anything so it works

Reply:
I know, I approved because I don't think this changes much.

Reply:
Yeah, I've addressed in 5bad13bd7a7eacb593df4408a59f1086b6a38569